integration/shareddata/composites.go

```diff
@@ -149,10 +163,10 @@ var ArrayStrings = &Values[string]{
 	},
 	data: map[string]any{
 		"array-string-desc":      bson.A{"c", "b", "a"},
-		"array-string-duplicate": bson.A{"b", "foo", "b"},
+		"array-string-duplicate": bson.A{nil, "foo", "b", "b", nil},
 		"array-string-numbers":   bson.A{"42", "0", "42.13"},
-		"array-string-nil":       nil,
```

Comment:
It's impossible to set `nil` value for an array in Tigris beta.27, so such examples are removed.

Reply:
Well, that also removes it for PostgreSQL, and that's bad

Reply:
Added a todo